cmd/kubeadm/app/cmd/token.go

```diff
@@ -213,8 +217,14 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
+	for _, group := range extraGroups {
```

Comment:
should probably only allow groups if usages has BootstrapTokenUsageAuthentication, right?